layouts/partials/partner-form/form.html

```diff
@@ -115,14 +115,14 @@ <h2>Great! Let's get started.</h2>
     </div>
 
     <div style="display:none" class="partner-form-how-long step active">
-      <label class="label">How long do you need the data to be stored on the network before you no longer need the data?</label>
+      <label class="label">How long do you need the data to be stored on the network before you no longer need the data?*</label>
```

Comment:
it is a good practice to bind label with corresponding input (type text, email or password) with 'for' attribute
or smth like this:
<label aria-label="mce-EMAIL-newsletter">
    <input class="subscribe-input" type="email" name="mce-EMAIL-newsletter" id="mce-EMAIL-newsletter" placeholder="Email Address" required="">
</label>